schema/charmcraft.json

```diff
@@ -57,7 +57,6 @@
 					],
 					"maxLength": 40,
 					"minLength": 1,
-					"pattern": "^([a-z0-9][a-z0-9-]?)*[a-z]+([a-z0-9-]?[a-z0-9])*$",
```

Comment:
question: did we forget to update the schema in the past and you just did it or is that an unrelated change?